mongo/test/org/immutables/mongo/fixture/JacksonRepoTest.java

```diff
@@ -79,11 +79,16 @@ public void setUp() throws Exception {
     module.addSerializer(new DateSerializer());
     module.addDeserializer(ObjectId.class, new ObjectIdDeserializer());
     module.addSerializer(new ObjectIdSerializer());
+    module.addDeserializer(UUID.class, new UUIDDeserializer(UuidRepresentation.JAVA_LEGACY));
 
     ObjectMapper mapper = new ObjectMapper()
-            .registerModule(JacksonCodecs.module(MongoClient.getDefaultCodecRegistry()))
-            .registerModule(new GuavaModule())
-            .registerModule(module);
+            .registerModule(JacksonCodecs.module(
+                CodecRegistries.fromRegistries(
```

Comment:
I noticed that my addition of `CodecRegistries.fromRegistries` here is unnecessary.

It was copied & pasted from a place where I include a custom UuidCodec, to configure `UuidRepresentation#STANDARD`